iphone/Maps/UI/Editor/MWMObjectsCategorySelectorDataSource.mm

```diff
@@ -43,6 +43,7 @@ - (void)load
 {
   m_categories = GetFramework().GetEditorCategories();
   m_categories.AddLanguage(locale());
+  m_categories.AddLanguage("en");
   auto const & types = m_categories.GetAllCreatableTypeNames();
   
   m_allCategories.reserve(types.size());
```

Comment:
OMG, you raised a problem with the Editor on iOS ..

- Why 2 separate lists m_allCategories and m_filteredCategories.
- getTypeIndex function is ill-formed
- Looks like when there is no type found, we show a full (sic!) list and it promotes "mistaging", I suspect.